src/ethereum_test_vm/tests/test_vm.py

```diff
@@ -383,6 +383,21 @@ def test_macros():
         pytest.param(
             Op.POP(Op.CALL(1, 2, 3, 4, 5, 6, 7)), 0, 0, 7, 0, id="POP(CALL(1, 2, 3, 4, 5, 6, 7))"
         ),
+        pytest.param(
```

Comment:
I added 5 cases, all the latter 4 ones fail because of the parenthesis. I'm not sure exactly which one of them is the most minimal, but I think it's reasonable to expect all of them to yield the same result, so let's test it.

I tried but couldn't reduce it further by taking away the instructions, but also didn't spend too much time on it, I think this is minimal enough.

Also, the original problem I encountered was actually mismatching and incorrect max stack height. It turns out that only the last two of these parenthesisations lead to max height also being invalid